apps/antalmanac/src/lib/tourExampleGeneration.ts

```diff
@@ -0,0 +1,243 @@
+                    startTime: {
+                        hour: 9,
+                        minute: 30,
+                    },
+                    endTime: {
+                        hour: 10,
+                        minute: 50,
+                    },
+                    timeIsTBA: false,
+                },
+            ],
+        },
+        {
+            meetings: [
+                {
+                    bldg: ['SSH 100'],
+                    days: 'MWF',
+                    startTime: {
+                        hour: 11,
+                        minute: 0,
+                    },
+                    endTime: {
+                        hour: 12,
+                        minute: 20,
+                    },
+                    timeIsTBA: false,
+                },
+            ],
+        },
+        {
+            meetings: [
+                {
+                    bldg: ['ALP 100'],
+                    days: 'TuTh',
+                    startTime: {
+                        hour: 11,
+                        minute: 0,
+                    },
+                    endTime: {
+                        hour: 11,
+                        minute: 50,
+                    },
+                    timeIsTBA: false,
+                },
+            ],
+        },
+    ];
+
+    const sampleClasses: Array<ScheduleCourse> = sampleClassesOptions.map(sampleClassFactory);
+
+    sampleClasses.forEach((sampleClass) => {
+        AppStore.addCourse(sampleClass);
+        sampleClassesSectionCodes.push(sampleClass.section.sectionCode);
+    });
+}
+
+export function removeSampleClasses() {
+    AppStore.deleteCourses(sampleClassesSectionCodes, CURRENT_TERM);
+    sampleClassesSectionCodes = [];
+}
+
+export function randint(min: number, max: number): number {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function randomWeekday(): DayOfWeek {
+    return daysOfWeek[randint(0, 6)];
+}
+
+function randomClasstime(): HourMinute {
+    return {
+        hour: randint(8, 17),
+        minute: randint(0, 5) * 10,
+    };
+}
+
+function randomStartEndTime(duration: number): [HourMinute, HourMinute] {
+    const start = randomClasstime();
+    const end = {
+        hour: start.hour + duration / 60,
+        minute: start.minute + (duration % 60),
+    };
+    return [start, end];
+}
+
+export function sampleMeetingsFactory({
+    bldg = ['DBH 1200'],
+    days = 'MWF',
+    startTime = {
+        hour: 11,
+        minute: 0,
+    },
+    endTime = {
+        hour: 11,
+        minute: 50,
+    },
+    timeIsTBA = false,
+}: Partial<WebsocSectionMeeting>): WebsocSectionMeeting[] {
+    return [
+        {
+            bldg,
+            days,
+            startTime,
+            endTime,
+            timeIsTBA,
+        },
+    ];
+}
+
+export function sampleFinalExamFactory({
+    examStatus = 'SCHEDULED_FINAL',
+    dayOfWeek,
+    month = 12,
+    day = 12,
+    startTime,
+    endTime,
+    bldg = [],
```

Comment:
```suggestion
    bldg = [DBH],
```
Have some building for the CourseCalendarEvent popup